content/release-10-17-0/digital-twin-manager-10-17-0-bundle/10-17-0-0.md

```diff
@@ -0,0 +1,223 @@
+---
+weight: 40
+title: Release 10.17.0.0
+layout: bundle
+---
+
+### Improvements
+
+<div><table ><colgroup>
+<col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>DTM application is now added with the <b>Home</b> page which mainly provides brief information about the application along with navigation to User guide. User can even use this page to navigate to <b>Asset properties, Asset models</b> and <b>Assets</b> creation page.</td>
+<td>CTM-311</td>
+</tr>
+
+<tr>
+<td><b>Assets</b> page now displays tree view representation of the assets for better visualize large hierarchies.</td>
+<td>CTM-128</td>
+</tr>
+
+<tr>
+<td>On click of an asset now user is provisioned to view the hierarchical/tree view representation of the asset under <b>Asset tree</b> tab. User can even expand and collapse the view using <b>Expand all</b> and <b>Collapse all</b> button.</td>
+<td>CTM-129</td>
+</tr>
+
+<tr>
+<td>Search feature is now added to the <b>Assets</b> page.</td>
+<td>CTM-227</td>
+</tr>
+
+<tr>
+<td>Default <b>Group</b> creation in <b>Asset models</b> grid is removed.</td>
+<td>CTM-388</td>
+</tr
+
+<tr>
+<td>Assets navigation from left navigation menu is no more supported.</td>
+<td>CTM-389</td>
+</tr
+
+<tr>
+<td>Assets can be added with child devices now.</td>
+<td>CTM-316</td>
+</tr>
+
+<tr>
+<td>Display of devices in the <b>Assets</b> and <b> Asset tree</b> page has been restricted.</td>
+<td>CTM-422</td>
+</tr>
+
+<tr>
+<td>Added a confirmation popup which states that the changes made to the <b>Asset hierarchy</b> page will be lost while leaving the page abruptly.</td>
+<td>CTM-249</td>
+</tr>
+
+<tr>
+<td>Added a hint which indicates the field is required while creating the asset.</td>
```

Comment:
Does the hint have text to indicate the field is required?

Reply:
Yes.. Example: <<AssetProperyName>>_(required)_